bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java

```diff
@@ -138,7 +139,10 @@ public void printPDF(final IXmlSerializable object, final PrintSettings printSet
          // setup XSL stylesheet source
          final StreamSource xslSource = new StreamSource(xslFile);
 
-         final Transformer transformer = TransformerFactory.newInstance().newTransformer(xslSource);
+         final TransformerFactory transformerFactory = TransformerFactory.newInstance();

```

Comment:
@FJBDev Can you fix this vulnerable XXE attack

Reply:
I believe it's thinking it's not fixed since I put UI.EMPTY_STRING instead of ""
When I had it originally with "", it was fine as it was addressing [the issue](https://rules.sonarsource.com/java/RSPEC-2755/) but it seems that it's not capable of seeing that UI.EMPTY_STRING is equivalent to ""

Reply:
The test failed

![image](https://user-images.githubusercontent.com/1283445/201032815-36bf9a42-8338-475c-a69a-b697e9b0006b.png)

Reply:
This test failure is something that I have fixed in this PR, it will now succeed when this is merged to main.
You can see [here](https://github.com/FJBDev/mytourbook/actions/runs/3429714365) an example of my repo's build and it is successful

Reply:
And notice the [new test report tab](https://github.com/FJBDev/mytourbook/runs/9392321694?check_suite_focus=true) that allows to quickly see the test results without having to look in the looonnngg build log 😄